.github/workflows/ci.yml

```diff
@@ -24,6 +33,7 @@ jobs:
         shell: bash
       - name: Test Templates
         uses: ./.github/actions/test-templates
+        if: runner.os != 'Windows'
```

Comment:
We can't test the templates on Windows?

Reply:
We need to update the action see https://github.com/icerpc/icerpc-csharp/issues/3976